components/fxa-client/src/fxa_client.udl

```diff
@@ -451,22 +451,24 @@ interface FirefoxAccount {
   //
   [Throws=FxaError]
   void set_push_subscription( DevicePushSubscription subscription );
-  
 
-  // Process and respond to server-delivered account update messages.
+
+  // Process and respond to a server-delivered account update message
   //
   // **💾 This method alters the persisted account state.**
   //
-  // Applications should call this method whenever they receive a push notiication on subscription
-  // endpoint previously registered with the Firefox Accounts server. Such messages typically indicate
-  // a noteworthy change of state on the user's account, such as an update to their profile information
-  // or the disconnection of a client. The [`FirefoxAccount`] struct will update its internl state
-  // accordingly and return a list of [`AccountEvent`] structs describing the events, which the application
+  // Applications should call this method whenever they receive a push notification from the Firefox Accounts server.
+  // Such messages typically indicate a noteworthy change of state on the user's account, such as an update to their profile information
+  // or the disconnection of a client. The [`FirefoxAccount`] struct will update its internal state
+  // accordingly and return an individual [`AccountEvent`] struct describing the event, which the application
   // may use for further processing.
   //
+  // It's important to note if the event is [`AccountEvent::CommandReceived`], the caller should immediately call
+  // [`FirefoxAccount::poll_device_commands`]
```

Comment:
nano doc nit: The "immediately" is over-emphasized here because the real point is something like "If the event is ..., the caller should not actually process the event - it should instead use the event information to notify the user, then arrange for `poll_device_commands()` to be called so that this, and any missed commands, are all correctly processed and the state of the commands correctly persisted to disk", or something like that?